server.py

```diff
@@ -1,37 +1,39 @@
 import json
 import traceback
+import os
+import rethinkdb as r
 
 from flask import Flask, jsonify, render_template, request, make_response
 
 import endpoints
 from utils.ratelimits import ratelimit
 
-app = Flask(__name__, template_folder='views', static_folder='views/assets')
+from dashboard import dash
 
+config = json.load(open('config.json'))
 
-def get_auth_keys():
-    try:
-        with open('keys.json') as keys:
-            data = json.load(keys)
-            if not isinstance(data, list):
-                print('keys.json must only contain an array of valid auth tokens')
-                return []
-            else:
-                return data
-    except FileNotFoundError:
-        print('keys.json wasn\'t found in the current directory')
-        return []
+RDB_ADDRESS = config['rdb_address']
+RDB_PORT = config['rdb_port']
+RDB_DB = config['rdb_db']
+
+rdb = r.connect(RDB_ADDRESS, RDB_PORT)

```

Comment:
`db=RDB_DB`, this removes the need to do `r.db(RDB_DB).table(...)` on each call.